capstone/src/main/webapp/script.js

```diff
@@ -162,6 +166,71 @@ function setUpReplacements(text, lang, substringForReplacements) {
   showDataMuseWordReplacementOptions(text, lang, substringForReplacements);
 }
 
+/** Gets the most extreme word replacements on a string */
+async function getExtremes(text, lang) {
```

Comment:
Let's break up this function with some helper methods.

Reply:
Done!